killerbee/dev_sewio.py

```diff
@@ -300,10 +326,37 @@ def set_channel(self, channel, page=0):
     # KillerBee expects the driver to implement this function
     def inject(self, packet, channel=None, count=1, delay=0, page=0):
         '''
-        Not implemented.
+        Injects the specified packet contents.
+        @type packet: String
+        @param packet: Packet contents to transmit, without FCS.
+        @type channel: Integer
+        @param channel: Sets the channel, optional
+        @type count: Integer
+        @param count: Transmits a specified number of frames, def=1
+        @type delay: Float
+        @param delay: Delay between each frame, def=0
+        @rtype: None
         '''
+
         self.capabilities.require(KBCapabilities.INJECT)
 
+        if len(packet) < 1:
+            raise Exception('Empty packet')
```

Comment:
`ValueError`s or similar?

Reply:
I just copied from your PR feel free to make any changes you want